patches/chrome-browser-resources-signin-profile_picker-profile_picker_app.ts.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/signin/profile_picker/profile_picker_app.ts b/chrome/browser/resources/signin/profile_picker/profile_picker_app.ts
+index 4138be7976b442c797e76d152e683d4d9da25635..c4179b55214a65b38cb5d1bc3dc40c0d21b4f9e9 100644
+--- a/chrome/browser/resources/signin/profile_picker/profile_picker_app.ts
++++ b/chrome/browser/resources/signin/profile_picker/profile_picker_app.ts
+@@ -179,6 +179,7 @@ export class ProfilePickerAppElement extends ProfilePickerAppElementBase {
+     } else {
+       this.shouldDisplayVerticalBanners_ = false;
+     }
++    this.shouldDisplayVerticalBanners_ = false;
```

Comment:
isn't whatever image they are using there part of branding so we can replace it?

Reply:
or if not, can we override the style using an existing patch?

Reply:
looks like it's not part of branding which is really odd, but a more extensible patch would be to replace it `static_files` `chrome/browser/resources/signin`